modules/troubleshooting-dspa-component-errors.adoc

```diff
@@ -0,0 +1,215 @@
+
+Ensure that your object store credentials and connection information are accurate, and verify that the object store is accessible from within the data science project's associated OpenShift namespace. One common issue is that the object storage SSL certificate is not trusted, particularly if self-signed certificates are used.
+
+Verify and update your object storage credentials, then retry the operation.
+
+|`ObjectStorageAvailable`
+
+Ready
+|False
+
+False
+|`Wrong credentials for Object Storage: Could not connect to (minio-my-project.apps.my-cluster.com), Error: The request signature we calculated does not match the signature you provided. Check your key and signing method.`
+|Provide the correct credentials for your object storage and retry the operation.
+
+|`DatabaseAvailable`
+
+Ready
+|False
+
+False
+|`FailingToDeploy: Dial tcp XXX.XX.XXX.XXX:3306 : i/o timeout`
+|If the issue persists beyond startup, check for network issues or misconfigurations in the database connection settings.
+
+|`DatabaseAvailable`
+
+Ready
+|False
+
+False
+|`Unable to connect to external database: tls: failed to verify certificate: x509: certificate signed by unknown authority`
+|This issue can occur when you use any external database, such as Amazon RDS. The data science pipelines manager cannot connect to the database because it does not trust the database SSL certificate, preventing the creation of the pipeline server.  Contact your IT operations administrator to add the relevant certificates.
+
+ifdef::upstream[]
+For more information, see link:{odhdocshome}/installing-open-data-hub/#understanding-certificates_certs[Understanding certificates in {productname-short}].
+endif::[]
+
+ifdef::self-managed[]
+ifndef::disconnected[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates].
+endif::[]
+
+ifdef::disconnected[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}_in_a_disconnected_environment/working-with-certificates_certs[Working with certificates].
+endif::[]
+endif::[]
+
+ifdef::cloud-service[]
+For more information, see link:{rhoaidocshome}{default-format-url}/installing_and_uninstalling_{url-productname-short}/working-with-certificates_certs[Working with certificates].
+endif::[]
+
+|`DatabaseAvailable`
+
+Ready
+|False
+
+False
+|`Error 1129: Host 'A.B.C.D' is blocked because of many connection errors.`
+| This issue can occur when you use any external database, such as Amazon RDS. The data science pipelines manager cannot connect to the database because it does not trust the database SSL certificate, preventing the creation of the pipeline server.  Contact your IT operations administrator to add the relevant certificates.
```

Comment:
According to the info in the related Google Docs, this part doesn't belong to this entry.

Reply:
Thanks, I have corrected this.